cmd/fleetctl/preview.go

```diff
@@ -350,7 +350,21 @@ func previewStopCommand() *cli.Command {
 			out, err := exec.Command("docker-compose", "stop").CombinedOutput()
 			if err != nil {
 				fmt.Println(string(out))
-				return errors.Errorf("Failed to run docker-compose stop")
+				return errors.Errorf("Failed to run docker-compose stop for Fleet server and dependencies")
+			}
+
+			cmd := exec.Command("docker-compose", "stop")
+			cmd.Dir = filepath.Join(previewDir, "osquery")
+			cmd.Env = append(cmd.Env,
+				// Note that these must be set even though they are unused while
+				// stopping because docker-compose will error otherwise.
+				"ENROLL_SECRET=empty",
+				"FLEET_URL=empty",
```

Comment:
I don't have a ton to add here, being new to the golang shebang, but as a casual reader, my interpretation is that "empty" here is equivalent to "unused" or "foo"

Reply:
Yeah, this is not very particular to go. Essentially we are trying to do the equivalent of this in bash
```
ENROLL_SECRET=empty FLEET_URL=empty docker-compose rm -sf
```

Reply:
Got it- so any meaningless non-empty string would be equivalent, since it’s there to fool some kind of “is truthy string” validation in Fleet’s docker-compose script/config.